### ISSUE TITLE
Settings: add Date and Time Formats selection option

### ISSUE THREAD

**Issue by wechsler42** [photoprism/photoprism#2107]
Hi there,
at the moment it seems that Photoprism uses the Date and Time Format of the GUI language chosen. For example:
a) Settings>>Language>>English

Time: h:mm (am/pm)
Date: DDD, MMM DD, YYYY

![image](https://user-images.githubusercontent.com/19371424/156575580-8fc244bd-011a-4a02-a797-a4966482a7cc.png)

b) Settings>>Language>>German

Time: hh:mm (24h clock)
Date: DDD, DD. MMM, YYYY

![image](https://user-images.githubusercontent.com/19371424/156575701-8e85610a-fa1e-44f3-b3c3-6af824ec5b15.png)

So I would like to suggest to add a Date and Time Format selection option to Settings:
Settings >>Date Format: (DDD, DD. MMM, YYYY//DDD, MMM DD, YYYY//...)
Settings >>Time Format: (h:mm//hh:mm//am/pm//24h//...)

Thanks for considering this suggestions.

**Comment by papatistos** [comment]
Indeed! Being able to set the time and date format would be excellent. Deducing the date format from the language doesn't work. USA and GB use different date formats...

**Comment by deployHuman** [comment]
Would also love this, I dont like having my native language in these apps, but then instead now i have to live with 12H clock ? that i hate even more :D

**Comment by PitRejection2359** [comment]
Yes please - I don't want my dates in US format  (why would you put the month (the middle "unit") at the start of the date!!) - I want my dates in a logical DD-MMM-YYYY format - smallest unit to largest unit!! :) Please can you fix this?!!
